test/test.go

```diff
@@ -28,6 +28,80 @@ func testEmbed(g abstract.Group, rand cipher.Stream, points *[]abstract.Point,
 	*points = append(*points, p)
 }
 
+func testPointSet(g abstract.Group, rand cipher.Stream) {
+	N := 1000
+	null := g.Point().Null()
+	for i := 0; i < N; i++ {
+		P1, _ := g.Point().Pick(nil, rand)
+		P2 := g.Point()
+		P2.Set(P1)
+		if !P1.Equal(P2) {
+			panic("Set() set to a different point.")
+		}
+		if !P1.Equal(null) {
```

Comment:
? Well, in every other situation I can still overlook the `if` also, that does not mean we don't prefer doing the regular code outside. I don't get why the `continue` keyword is an exception ... it's part of the language as well.
But ok, I'm only back a few hours from my vacation here, so I don't really mind :D